prow/config/secret/secret.go

```diff
@@ -42,5 +47,11 @@ func LoadSingleSecret(path string) ([]byte, error) {
 	if err != nil {
 		return nil, fmt.Errorf("error reading %s: %v", path, err)
 	}
-	return bytes.TrimSpace(b), nil
+
+	secret := bytes.TrimSpace(b)
+
+	if !tokenRegex.Match(secret) {
+		return nil, fmt.Errorf("error reading %s: invalid token format", path)
```

Comment:
Another nit: Could you print the regex here, someting like `invalid token format: Does not match regex \`^...\``?